src/Mod/Path/PathScripts/PathUtils.py

```diff
@@ -231,31 +162,42 @@ def horizontalFaceLoop(obj, face, faceList=None):
         # wire is still closed and it still has the same footprint
         bb1 = comp.BoundBox
         bb2 = w.BoundBox
-        if w.isClosed() and PathGeom.isRoughly(bb1.XMin, bb2.XMin) and PathGeom.isRoughly(bb1.XMax, bb2.XMax) and PathGeom.isRoughly(bb1.YMin, bb2.YMin) and PathGeom.isRoughly(bb1.YMax, bb2.YMax):
+        if (
+            w.isClosed()
+            and PathGeom.isRoughly(bb1.XMin, bb2.XMin)
+            and PathGeom.isRoughly(bb1.XMax, bb2.XMax)
+            and PathGeom.isRoughly(bb1.YMin, bb2.YMin)
+            and PathGeom.isRoughly(bb1.YMax, bb2.YMax)
+        ):
             return faces
     return None
 
 
 def filterArcs(arcEdge):
-    '''filterArcs(Edge) -used to split arcs that over 180 degrees. Returns list '''
+    """filterArcs(Edge) -used to split arcs that over 180 degrees. Returns list"""
     PathLog.track()
     s = arcEdge
     if isinstance(s.Curve, Part.Circle):
         splitlist = []
         angle = abs(s.LastParameter - s.FirstParameter)
         # overhalfcircle = False
         goodarc = False
-        if (angle > math.pi):
+        if angle > math.pi:
```

Comment:
What does `angle` contain? The angle in radians or which units? If the angle bigger than 3.14, I don't currently understand

Reply:
Good point.  I've added a test case for filterArcs and refactored this function to be more readable.
Caught an error in the process!